app/utils/scheduler.py

```
@@ -1614,397 +1629,410 @@ async def _execute_auto_buy(self, send_slack_notification: bool = True):
+                if execution_tasks:
+                    try:
+                        db = get_db()
+                        if db is not None:
+                            # 최근 5분 이내에 체결 완료된 매수 주문 수 확인
+                            five_minutes_ago = datetime.now() - timedelta(minutes=5)
+                            executed_count = db.trading_logs.count_documents({
+                                "order_type": "buy",
+                                "status": "executed",
+                                "created_at": {"$gte": five_minutes_ago}
+                            })
+                    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)